### PR TITLE
Addition of latest posts

### PR DESCRIPTION
**Pull request by jasonwilliams**
- In order for latest posts to work it needed to be a plugin which overrides the blog plugin.
- This also means that the home page can't be a regular page but is instead loaded from the blog plugin and has the "recent posts" injected
- Readme should refernce npm as we're using packahe-lock.json
- Prettier has changed quotes (we should fix that)

image

### CODE REVIEW

File: src/components/latestPosts/latestPosts.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import clsx from "clsx";
+import React from "react";
+import styles from "./styles.module.css";
+
+export default function LatestPosts({ recentPosts }) {
+  return (
+    <div className={clsx("container", styles.wrap)}>
```
Comment (nekevss) [inline src/components/latestPosts/latestPosts.tsx:8]: Would it be worthwhile using Infima's card component for the blog posts over a list.

**Review by nekevss — APPROVED** [review]
This looks fantastic! I was curious about what you thought about using Infima's cards over a list to make the posts a bit more engaging (comment below), but I'd be game with merging either way.

File: src/components/latestPosts/latestPosts.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      <h2>Latest posts and feature explainers</h2>
```
Comment (jedel1043) [inline src/components/latestPosts/latestPosts.tsx]:
```suggestion
      <h2>Latest posts</h2>
```
Just for simplicity.

**Review by jedel1043 — APPROVED** [review]
Really liking how the new web page is coming along!

**Comment by jasonwilliams** [comment]
Thanks for the feedback, I should have done this as a draft, I didn’t mean it to be a final solution, just to show what’s been done so far. I will take a look at the cards idea as that could work, I assume this is built in?

**Comment by nekevss** [comment]
Yepp, from what I can tell. The infima docs are [here](https://infima.dev/), and I used them somewhat heavily in the conformance components.

**Comment by jasonwilliams** [comment]
Some updates:

image


image

**Pull request merged**